beets/plugins.py

```diff
@@ -488,3 +488,13 @@ def sanitize_choices(choices, choices_all):
             if not (s in seen or seen.add(s)):
                 res.extend(list(others) if s == '*' else [s])
     return res
+
+
+def notify_info_received(msg):
```

Comment:
The new decorator could probably use a docstring. Also, the functionality here doesn't seem specific to the `*_info_received` events—so perhaps the name should be more general too?